### PR TITLE
Fix adding framework targets to AppClip 

### PR DESCRIPTION
**Pull request by sampettersson**
### Short description 📝

`validLinks` wasn't defined for `.appClip` ending in an error `Target A has a dependency with target B of type framework for platform 'iOS' which is invalid or not supported yet.` this PR resolves that.

### Checklist ✅

- [x] The code architecture and patterns are consistent with the rest of the codebase.
- [x] The changes have been tested following the [documented guidelines](https://tuist.io/docs/contribution/testing-strategy/).
- [x] The `CHANGELOG.md` has been updated to reflect the changes. In case of a breaking change, it's been flagged as such.
- [x] In case the PR introduces changes that affect users, the documentation has been updated.

### CODE REVIEW

**Review by pepibumur — APPROVED** [review]
Thanks for fixing this @sampettersson. Would you mind updating the CHANGELOG?

**Comment by pepibumur** [comment]
@all-contributors add @sampettersson for code

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 ### Changed
 
 - Replace `ExpressibleByStringLiteral` with `ExpressibleByStringInterpolation` for `ProjectDescription` objects by [@DimaMishchenko](https://github.com/DimaMishchenko)
+- Fix adding framework targets to AppClip by [@sampettersson](https://github.com/sampettersson)
```
Comment (fortmarek) [inline CHANGELOG.md:14]: add a link to the PR please (see below) => I think the log above does not have a PR link by mistake

**Review by fortmarek — APPROVED** [review]
looks good! Try to rebase main, there are some unrelated acceptance tests failing. Ping me here if it fails even after rebase but since the main is green, it should work fine.

**Comment by pepibumur** [comment]
@all-contributors add @sampettersson for code

**Pull request merged**